### ISSUE TITLE
Security - CVE-2017-17485 & CVE-2018-5968 on Jackson dependency 

### ISSUE THREAD

**Issue by cdanger** [bguerout/jongo#327]
Hello,
running owasp dependendency-check on a project using jongo will cause the error below because jongo uses a version of jackon-databind affected by CVE-2017-17485 & CVE-2018-5968.

```
[ERROR] Failed to execute goal org.owasp:dependency-check-maven:3.0.2:check (default) on project authzforce-ce-core-pdp-testutils: 
[ERROR] 
[ERROR] One or more dependencies were identified with vulnerabilities: 
[ERROR] 
[ERROR] jackson-databind-2.7.3.jar: CVE-2017-17485, CVE-2018-5968
```

**Comment by cdanger** [comment]
see the mentioned pull request for the fix

**Comment by cdanger** [comment]
Hello, I can see that the fixing PR #326 is now merged and planned to be part of milestone 1.4.0, but there are still a few open issues planned as well, blocking the release. Could you to do a hotfix release in the meantime (to have this security fix part of the release)? Thank you.
